README.md

```diff
@@ -58,60 +96,124 @@ This deployment is available under the _kafka-inmemory_ folder and provides foll
+cluster as service `kafka-connect` on port `8083`.
 
-### Deploying to OpenShift
 
-To conveniently deploy Kafka Connect to OpenShift a [Template is provided](kafka-connect/resources/openshift-template.yaml), this could be used via `oc create -f kafka-connect/resources/openshift-template.yaml` so it is present within your project. To create a whole Kafka Connect cluster use `oc new-app barnabas-connect` *(make sure you have already deployed a Kafka broker)*.
+### Deploying to OpenShift Origin
+
+1. Deploy a Kafka broker to your OpenShift cluster using either of the [in-memory](#kafka-in-memory) or 
+   [statefulsets deployments](#kafka-stateful-sets) above.
+2. Create a pod using the [provided template](kafka-connect/resources/openshift-template.yaml) by running
+
+        oc create -f kafka-connect/resources/openshift-template.yaml
+
+   in your terminal.
+3. Create a new app:
+
+        oc new-app barnabas-connect
+
 
 ### Deploying to Kubernetes
 
-To conveniently deploy Kafka Connect to Kubernetes use the provided yaml files with the Kafka Connect [deployment](kafka-connect/resources/kafka-connect.yaml) and [service](kafka-connect/resources/kafka-connect-service.yaml). This could be done using `kubectl apply -f kafka-connect/resources/kafka-connect.yaml` and `kubectl apply -f kafka-connect/resources/kafka-connect-service.yaml`.
+1. Deploy a Kafka broker to your Kubernetes cluster using either of the [in-memory](#kafka-in-memory) or 
+   [statefulsets deployments](#kafka-stateful-sets) above.
+2. Start the deployment by running 
+
+        kubectl apply -f kafka-connect/resources/kafka-connect.yaml
+
+   in your terminal.
+3. Start the Connect service by running
+
+        kubectl apply -f kafka-connect/resources/kafka-connect-service.yaml
+
+   in your terminal.
+
+4. Top tear down the services run:
+
+        kubectl delete -f kafka-connect/resources/kafka-connect-service.yaml && \
+        kubectl delete -f kafka-connect/resources/kafka-connect.yaml
+
 
 ### Using Kafka Connect with additional plugins
 
 Our Kafka Connect images contain by default only the `FileStreamSinkConnector` and `FileStreamSourceConnector` connectors.
 If you want to use other connectors, you can:
+
 * Mount a volume containing the plugins to path `/opt/kafka/plugins/`
-* Use the `enmasseproject/kafka-connect` image as Docker base image, add your connectors to the `/opt/kafka/plugins/` directory and use this new image instead of `enmasseproject/kafka-connect`
+* Use the `enmasseproject/kafka-connect` image as Docker base image, add your connectors to the `/opt/kafka/plugins/` 
+  directory and use this new image instead of `enmasseproject/kafka-connect`
 * Use OpenShift build system and our S2I image
 
 #### Mount a volume containing the plugins
 
-* Prepare a persistent volume which contains a directory with your plugin(s).
-* Mount the volume into your Pod
+1. Prepare a persistent volume which contains a directory with your plugin(s).
```

Comment:
I think we should maybe add something like *Set the permissions to the plugin files so that they can be read by all users*